Dockerfile

```diff
@@ -7,10 +7,8 @@ RUN mkdir -p /code && \
                 python3-libmodulemd1 modulemd-tools python-gobject git -y && \
     yum clean all
 RUN curl https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -o wait_for_it.sh && chmod +x wait_for_it.sh
-COPY requirements.txt /tmp/requirements.txt
+COPY requirements.txt /code/requirements.txt
 RUN cd /code && virtualenv -p python3.9 --system-site-packages env && source env/bin/activate \
-    && pip3 install --upgrade pip && pip3 install -r /tmp/requirements.txt --no-cache-dir
-COPY alws /code/alws
-COPY tests /code/tests
+    && pip3 install --upgrade pip && pip3 install -r /code/requirements.txt --no-cache-dir
 WORKDIR /code
-CMD ["/bin/bash", "-c", "source env/bin/activate && uvicorn --workers 4 --host 0.0.0.0 alws.app:app"]
+CMD ["/bin/bash", "-c", "source env/bin/activate && pip3 install --upgrade pip && pip3 install -r requirements.txt --no-cache-dir && uvicorn --workers 4 --host 0.0.0.0 alws.app:app"]
```

Comment:
what a purpose of installing dependencies in entrypoint if they were installed in previous layers?

Reply:
We pre-build all images and req.txt is single item that's used inside an image. So, we need reinstall requirements if a developer changes them. That's why I mount req.txt to a container, because it can be changed by compare with the internal version in an image.